src/jsonrpc-discover.json

```diff
@@ -34,6 +34,27 @@
       }
     },
 
+    {
+      "name": "rebind",
+      "summary": "Rebinds the server",
+      "params": [ {
+          "name":"port",
```

Comment:
I think this should be an address, just like what the call to fork returns.

Reply:
Ok, I changed the API to use `address` argument, just like the one returned by fork.